presto-parser/src/main/java/com/facebook/presto/sql/SqlFormatter.java

```diff
@@ -888,4 +903,23 @@ private static void appendAliasColumns(StringBuilder builder, List<String> colum
             builder.append(')');
         }
     }
+
+    public static String getFormattedSql(Statement statement, SqlParser sqlParser)
+    {
+        String sql = formatSql(statement);
+
+        // verify round-trip
+        Statement parsed;
+        try {
+            parsed = sqlParser.createStatement(sql);
+        }
+        catch (ParsingException e) {
+            throw new ParsingException("Formatted query does not parse: " + statement);
```

Comment:
PrestoException is not accessible from here without creating a dependency from presto-parser on presto-spi.  Is there a different exception I should use, or is it reasonable to create that dependency?